pkg/sif/lookup.go

```diff
@@ -254,6 +254,11 @@ func (descr *Descriptor) GetData(fimg *FileImage) []byte {
 		return data
 	}
 
+	if int64(len(fimg.Filedata)) < descr.Fileoff+descr.Filelen {
```

Comment:
This is pure nitpicking, but I find this easier to read:

```suggestion
	if descr.Fileoff+descr.Filelen > int64(len(fimg.Filedata)) {
```